Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -122,9 +144,60 @@ def Druva_FindDevice_Command(clientObj, search_string):
     else:
         raise RuntimeError('Error: ' + str(response.status_code))
 
+def Druva_FindUser_Command(clientObj, user_string):
+    response = clientObj.get_findUser(user_string)
+    statusCode = response.status_code
+    if (statusCode==200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Found Druva users', responseJson['users'])
+        outputs = {"Druva.User(val.userID == obj.userID)": responseJson['users']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
```

Comment:
Lets change this to a commandResults object. You can see [this article](https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults). Relevant to all the new commands added

Reply:
commandResults object is used in all commands.